manifest-generator/generator-services/src/test/resources/plugins/health/healthcheck-http.hspec

```diff
@@ -0,0 +1,38 @@
+name: myservice
+image:
+  registry: x.y.z
+  name: myServiceImage
+  tag: 1.2.3
+  buildSpec:
+    stackImage: abc/pqr:1.0
+    artifacts:
+      - name: sample
+        source: sample.war
+        destination: /abc/def/ghi/sample.txt
+    configCommands: |-
+      echo "Hello, I'm configure commands"
+    runCommandsScript: /abc/def/run.sh
+    runCommands: |-
+      echo "Hello, I'm run commands"
+
+
+startCommand: "a,bc,def"
+replicas: 
+  min: 1
+  max: 3
+  cpuThreshold: 10%
+memory: 123Mi-456Mi
+cpu: 123m-456m
+
+volumes:
+  - name: vol-name
+    path: /volume/mount/path
+    size: 1Gi
+    storageClass: storageClass
+
+external: true
+ports:
+  - port: 8080/http
```

Comment:
Do we need separate hspec for each case ? Can we rely on a generic hspec for postive cases mainly ?